src/test/java/bkg/b2/bookingSummariesIT.java

```diff
@@ -0,0 +1,91 @@
+package bkg.b2;
```

Comment:
I assume this is supposed to be v2?

Reply:
i went with b2 due to the beta naming, but yeah i guess V2 makes more sense

Reply:
I'm unsure if it's v/b1 or 2. Swagger is v1 for BKG and v2 for eBL.

Reply:
no BKG is version 1 so it should go into v1

Reply:
Done